datapacks/Bookshelf/data/bs.schedule/functions/cancel_all.mcfunction

```diff
@@ -0,0 +1,20 @@
+# INFO ------------------------------------------------------------------------
+# Copyright © 2023 Gunivers Community.
+
+# Authors: theogiraudet, Aksiome
+# Contributors:
+
+# Version: 2.0
```

Comment:
The remove_all feature didn't exist before the PR so this is the ﬁrst version if this feature.